src/components/BarChart.vue

```diff
@@ -0,0 +1,83 @@
+<template>
+  <BarChart
+    v-bind="barChartProps"
+    :style="{
+      width: '15rem',
+      height: '15rem',
+      alignItems: 'center',
+    }"
+  />
+</template>
+
+<script>
+import { computed, defineComponent, ref, toRefs } from "vue";
+import { BarChart, useBarChart } from "vue-chart-3";
+import { Chart, registerables } from "chart.js";
+
+const dataChange = (data) => {
+  let arr = [];
```

Comment:
```suggestion
  const arr = [];
```
arr은 재할당하지 않으니 const로 선언하는게 좋을 듯 합니다.